content/_code-samples/xrpl4j/GetAccountInfo.java

```diff
@@ -0,0 +1,25 @@
+// Construct a network client
```

Comment:
I'm not against this, although a sample `pom.xml` file wouldn't be as useful as a `package.json` file, since maven requires a certain directory structure.  

I guess I could copy the entire maven project into the `content/_code-samples/xrpl4j` directory for easier startup, but linking to the external repo seems sufficient?